blueocean-analytics-tools/src/main/resources/io/jenkins/blueocean/analyticstools/AnalyticsTools/header.jelly

```diff
@@ -0,0 +1,32 @@
+          ]]>
+        </script>
```

Comment:
@tfennelly I just moved this code from under blueocean-web over here. I will let others, dig thru rollbar and see how to iify it.

I will update index.jelly description though.